src/Data/Ticket/TicketDataClass.ts

```diff
@@ -8,7 +8,15 @@ export const Ticket = provideLocalStorageDataClass('Ticket', {
 
     return {
       ...newData,
-      status: newData.status || 'captured',
+      type:
+        newData.type || newData.typeLocalized === 'Question'
+          ? 'PSA_SVC_CAL'
+          : newData.typeLocalized === 'Claim'
+            ? 'PSA_SVC_CPL'
+            : 'PSA_SVC_TRB',
```

Comment:
* [ ] FYI complex frontend logic a at least a yellow, if not a red flag. Every consumer has to agree on the same logic instead of "just consuming" the backend. This may introduce subtle errors which cannot be fixed by just fixing the backend. Should we put a comment like "can be removed when the backend is fixed" here?

Reply:
This code is only intended as a quickfix for the localstorage to work. With a real backend you would not use such code.